src/docfx/lib/HtmlUtility.cs

```diff
@@ -171,7 +171,7 @@ private static void AddLinkType(this HtmlNode html, string tag, string attribute
                     node.SetAttributeValue("data-linktype", "self-bookmark");
                     continue;
                 }
-                if (href[0] == '/' || href[0] == '\\')
+                if ((href[0] == '/' || href[0] == '\\') && !href.StartsWith("//"))
```

Comment:
>!href.StartsWith("//") [](start = 59, length = 22)

put `external` check before this and remove this?

Reply:
IsUrl treats `/asdf` as a valid url